oomd/codegen/oomd.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
-// 	protoc        v3.17.3
+// 	protoc-gen-go v1.26.0
```

Comment:
any chance we have more up-to-date versions?

Reply:
Unless we modify and maintain this image

Reply:
nvm, leave them as is